LoopKit/Persistence/Model.xcdatamodeld/Modelv2.xcdatamodel/contents

```diff
@@ -2,11 +2,11 @@
 <model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="16119" systemVersion="19G73" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
     <entity name="CachedCarbObject" representedClassName=".CachedCarbObject" syncable="YES">
         <attribute name="absorptionTime" optional="YES" attributeType="Double" usesScalarValueType="NO"/>
+        <attribute name="anchorKey" attributeType="Integer 64" defaultValueString="0" usesScalarValueType="YES" elementID="modificationCounter"/>
```

Comment:
👍   Just noting that I checked: CoreData auto-migration supports renaming a property (https://developer.apple.com/documentation/coredata/using_lightweight_migration). (Though... I'm not 100% sure how it "knows" this is a rename given only this `.xcdatamodel` file, but I won't worry too much about that...)

Reply:
Yep. Lightweight migration uses the `elementID` field (above set to `"modificationCounter"`), which in the Xcode UI is called the "renamingIdentifier".

Reply:
Ahhh